wagtailio/features/blocks.py

```diff
@@ -39,9 +39,37 @@ class Meta:
         template = "features/blocks/additional_features_block.html"
 
 
+class FeatureBlock(blocks.StructBlock):
+    heading = blocks.CharBlock(label="Feature heading", required=False, max_length=255)
+    description = blocks.RichTextBlock(
+        label="Feature description",
+        required=True,
+        features=["bold", "italic", "link", "document"],
+    )
+    link = blocks.URLBlock(label="Feature link", required=False)
```

Comment:
Note: I think we'll need another field here. The design has "view docs" which look like it could be hard-coded but what happens if we're not linking to the docs?

Reply:
To be clear, we'll need a button title field

Reply:
Thanks, updated in 1a36be7